Source/Core/ShoKanri.Domain/Contracts/Data/Repositories/User/IUserWriteRepository.cs

```diff
@@ -2,7 +2,4 @@
 
 namespace ShoKanri.Domain.Contracts.Data.Repositories.User;
 
-public interface IUserWriteRepository : IWriteOnlyRepository<Entities.User>
-{
-    Task DeleteAsync(Entities.User user);
-}
+public interface IUserWriteRepository : IWriteOnlyRepository<Entities.User> {}
```

Comment:
```suggestion
public interface IUserWriteRepository : IWriteOnlyRepository<Entities.User>;
```

Para deixar mais limpo.